src/Workspaces/CSharp/Portable/Extensions/SemanticModelExtensions.cs

```diff
@@ -313,7 +313,7 @@ private static IList<ParameterName> GenerateNames(IList<string> reservedNames, L
         public static ISet<INamespaceSymbol> GetUsingNamespacesInScope(this SemanticModel semanticModel, SyntaxNode location)
         {
             // Avoiding linq here for perf reasons. This is used heavily in the AddImport service
-            HashSet<INamespaceSymbol> result = null;
+            var result = new HashSet<INamespaceSymbol>();
```

Comment:
Oh.  Hmm.  I suspect we might run into problems with merged namespaces, if I declare myself inside `namespace System`, but also have references to assemblies which define `System`.  Do we need a different kind of equality here?